exporter/datadogexporter/examples/collector.yaml

```diff
@@ -255,10 +273,25 @@ exporters:
       #
       # tags: []
 
-
+# `service` defines the Collector pipelines, observability settings and extensions.
 service:
+  # `telemetry` allows you to customize the Collector's own telemetry.
+  telemetry:
+    logs:
+      # Set the log level to info (the default)
+      level: info
```

Comment:
I would remove this since this is the default
```suggestion
```

Reply:
Agreed. There is no gain in documenting OTel options. The purpose here should be to underline Datadog exporter config, without any extra noise.